File: Estrutura_Sequencial/Java/Ex1002/Main.java
package Ex1002;

import java.util.Scanner;

public class Main {
    @SuppressWarnings("resource")
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double raio = sc.nextDouble();
        double n = 3.14159;
        double area = n * Math.pow(raio, 2);

        System.out.printf("A=%.4f\n", area);
    }
}

File: Estrutura_Sequencial/C++/Ex1002/main.cpp
#include <iostream>
#include <math.h>
#include <iomanip>
#define n 3.14159

int main(){
    double raio, area;


    std::cin >> raio;
    area = n * pow(raio, 2);

    std::cout << std::fixed << std::setprecision(4);
    std::cout << "A=" << area << std::endl;

    return 0;
}